src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/Contact.php

```diff
@@ -168,4 +168,14 @@ public function isContactSet(): bool
     {
         return !(empty($this->firstName) && empty($this->lastName) && empty($this->email) && empty($this->phone));
     }
+
+    public function isSet()
+    {
+        $isNull = $this->email === null &&
+        $this->firstName === null &&
```

Comment:
Indent 3 lines to make it more readable.

Reply:
This looks to be a regression from the previous PR. the isSet method was removed in favor of the already present 'isContactSet'. Will remove this entirily